### ISSUE TITLE
Develop test case: One-off search feature does not affect search engine preferences after updating Firefox.

### ISSUE THREAD

**Issue by tracywalker** [mozilla/iris#224]
https://testrail.stage.mozaws.net/index.php?/cases/view/108269

**Comment by icrisan** [comment]
fix can be found in branch 'issue_224_1189'. @tracywalker please review. Thank you!
p.s: 
this is an update test case for the awesomebar section. 
test works(images are correct) for Firefox upgrade from 61.0b9 version to the latest one(63.0b8).
test can be run like this: iris -t one_off_preferences_persit_after_firefox_update -f 61.0b9 -o

**Comment by tracywalker** [comment]
please correct the typo in the test name. _persit_ should be _persist_.
This test cases is also missing image captures for search_engine_61.0b9.png on Linux and Windows.   
The test case PASSES on Mac.

**Comment by icrisan** [comment]
Hi Tracy! The missing image from Windows and Linux is not necessary because
we can use in both Firefox versions the same
pattern(search_engine_pattern). Does the test fail on Windows and Linux?

**Comment by tracywalker** [comment]
Yes, Fails on both Windows and Linux trying to get the search_engine_61.0b9.png which doesn't exist in Linux and Windows images folders.  I see what is going. The search_engine_custom_pattern = Pattern('search_engine_61.0b9.png') should be for Mac only.  I'll fix it and the test name typo.  Thanks Ioana.

**Comment by icrisan** [comment]
Thank you Tracy!

**Comment by tracywalker** [comment]
code approved
